challenge/src/components/Pages/About.jsx

```diff
@@ -25,29 +40,39 @@ const About = (props) => {
                 height: "100%",
                 alignItems: "center",
                 justifyContent: "center",
-                flexDirection: "row"
+                flexDirection: "row",
+                flexWrap: "wrap",
               }}
               >
-              <RandomImage
-                  page={page * 5}
-                  style={pictureStyling}
-                />
-              <RandomImage
-                  page={page * 9}
-                  style={pictureStyling}
-                />
-              <RandomImage
-                  page={page * 13}
-                  style={pictureStyling}
-                />
-                
+              <JojoStand
```

Comment:
this is very close to being modular, perhaps we could make an array like `[{stand: "The World", url: "https://..."}]`
and then you can use .map to loop through this and render a component for each object in this array.
kinda like `{stands.map((stand, i) => <JojoStand stand={stand.stand}>)}`